string mittente
string destinatario
---
string responseControl
